app/Http/Resources/VideosResource.php

```diff
@@ -31,6 +31,7 @@ public function toArray(Request $request): array
                 'call_to_action_link' => $item->call_to_action_link,
                 'creator' => $item->creator?->name,
                 'rating' => $item->proposal?->film_rating,
+                'sponsored_by' => $item->sponsor,
```

Comment:
@renjeroham I think that since sponsor is a relation, you will probably need the `$item->sponsor->name` here. Right?